internal/bminventory/inventory.go

```diff
@@ -2975,9 +2951,7 @@ func shouldStepReplyBeLogged(reply *models.StepReply, host *common.Host) bool {
 	// Here we are filtering these errors and log the message once per host.
 	// TODO: Create a new state "unstable" in the state machine with no commands.
```

Comment:
Looks like you even suggested to add a new state here

Reply:
That was also my position, but it has been decided

Reply:
Okay. so please update this comment (the TODO) and add a note above the * -> Disconnected transitions that explains that we're re-using the disconnected state to also mean "Media Disconnection"

Reply:
sm.AddTransition(stateswitch.TransitionRule{
		TransitionType: **TransitionTypeMediaDisconnect**,
		SourceStates: []stateswitch.State{
			stateswitch.State(models.HostStatusDiscovering),
			stateswitch.State(models.HostStatusInsufficient),
			stateswitch.State(models.HostStatusKnown),
			stateswitch.State(models.HostStatusPendingForInput),
			stateswitch.State(models.HostStatusDisconnected),
			stateswitch.State(models.HostStatusBinding),
		},
		DestinationState: stateswitch.State(models.HostStatusDisconnected),
		PostTransition:   th.PostHostMediaDisconnected,
	})

Reply:
The transition type is: TransitionTypeMediaDisconnect

Reply:
I know that, I still think you should have a comment there as overloading the Disconnected state to mean new things is not too obvious. Anyway, doesn't matter too much

Reply:
All the transitions to disconnected named TransitionTypeMediaDisconnect. 
The transition from disconnected to another status has a different condition (IsMediaConnected) from readability